lib/plist-transform.js

```diff
@@ -31,9 +32,29 @@ module.exports = (dir, pkg) => {
 		.then(content => {
 			const data = fix(plist.parse(content));
 			data.version = pkg.version || '';
-			data.description = pkg.description || '';
-			data.webaddress = pkg.homepage || pkg.author.url || '';
-			data.createdby = pkg.author.name || '';
+			data.description = data.description || pkg.description || '';
```

Comment:
I see what you mean, so you think `description`, `createdby`, and `webaddress`  (`version` already is) should be forced from `package.json`?